Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy+PathComponent.swift

```diff
@@ -243,6 +246,65 @@ private struct PathComponentScanner {
         return scanUntilSeparatorAndThenSkipIt()
     }
     
+    mutating func _scanOperatorName() -> Substring? {
```

Comment:
Should this be private?

Reply:
It can't be. I extracted this code so that `PathParser.parseOperatorName(_:)` (a few lines above) could call it.

The entire `PathComponentScanner` type is file private, so this is also implicitly file private.